descriptor.go

```diff
@@ -0,0 +1,25 @@
+package oras
+
+import (
+	"fmt"
+
+	"github.com/opencontainers/go-digest"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+// GenerateDescriptor returns an OCI descriptor, given the content and media type.
+func GenerateDescriptor(content []byte, mediaType string) (ocispec.Descriptor, error) {
+	if mediaType == "" {
+		return ocispec.Descriptor{}, fmt.Errorf("missing media type")
+	}
+	return ocispec.Descriptor{
+		MediaType: mediaType,
+		Digest:    digest.FromBytes(content),
+		Size:      int64(len(content)),
+	}, nil
+}
+
+// Equal returns true if two OCI descriptors point to the same content.
```

Comment:
I think we can omit the word "OCI" since there will only be OCI descriptors going forward.

Reply:
Done.